source/blog/2016-07-08-release-notes-6-36.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: Finagle 6.36 Release Notes
+published: true
+post_author:
+  display_name: Vladimir Kostyukov
+  twitter: vkostyukov
+tags: Releases, Finagle, Util, Ostrich, Scrooge, TwitterServer
+---
+
+This is Finagle's first release only for Java 8 and Scala 2.11. Since the previous release, the team
+has been focusing on the Netty 4 migration as well as eliminating some technical debt.
+
+### Towards Netty 4
+
+The team is firing on all cylinders towards Netty 4 support in Finagle. This milestone brings
+*beta* (not yet widely used in production) Netty 4 support to Mux/ThriftMux, HTTP 1.1, and
+SOCKS5 (proxy).
+
+In the meantime, we're part way through migrating MySQL, Memcached, and Redis protocol. Everything
```

Comment:
s/migrating MySQL, Memcached, and Redis protocol/migrating the MySQL, Memcached, and Redis protocols